libs/langchain-aws/src/common.ts

```diff
@@ -82,7 +82,26 @@ export function convertToConverseMessages(messages: BaseMessage[]): {
     .map((msg) => {
       if (msg._getType() === "ai") {
         const castMsg = msg as AIMessage;
-        if (typeof castMsg.content === "string" && castMsg.content !== "") {
+        if (castMsg.tool_calls && castMsg.tool_calls.length) {
```

Comment:
Instead of mapping over it, calling `.reduce` would likely be a better method, so we can keep all content fields. Could you push up this refactor? If not I can push up in a little